lib/src/log_manager.dart

```diff
@@ -107,6 +109,11 @@ class LogManager {
   /// Note: ideally this will be replaced w/ inline registrations within the
   /// flutter foundation binding (see e.g., https://github.com/flutter/flutter/pull/21505).
   void initServiceExtensions() {
+    // Avoid double initialization.
+    if (_initialized == true) {
```

Comment:
...and then just check `if (_initialized)` here.

Reply:
... but this check was failing.  Chalk it up to another nuance in Dart initialization rules but I was hitting an NPE here as `_initialized` was being seen as `null`.  (FWIW Janice was equally flummoxed!)